src/wallet/KintoWallet.sol

```diff
@@ -324,12 +328,9 @@ contract KintoWallet is Initializable, BaseAccount, TokenCallbackHandler, IKinto
             (
                 signerPolicy == SINGLE_SIGNER && owners.length == 1
                     && _verifySingleSignature(owners[0], hashData, userOp.signature) == SIG_VALIDATION_SUCCESS
-            )
-                || (
-                    signerPolicy != SINGLE_SIGNER
-                        && _verifyMultipleSignatures(hashData, userOp.signature) == SIG_VALIDATION_SUCCESS
-                )
+            ) || (signerPolicy != SINGLE_SIGNER && _verifyMultipleSignatures(hashData, userOp.signature))
```

Comment:
I think you still need the == SIG no? need to check if is euqls to 0 otherwise is the opopsite

Reply:
No, because I return bool in `_verifyMultipleSignatures`

Reply:
sounds good, or correct is false got it